### DISCUSSION TITLE
Draw.io for Jira Software: Templates under New Diagram available?

### DISCUSSION THREAD

**Discussion by Preservation-Quest** [jgraph/drawio#2496] [Q&A]
Greetings Draw.io and co,

I've installed and am using Draw.io for Jira Data Center (https://marketplace.atlassian.com/apps/1211413/draw-io-diagrams-for-jira?tab=overview&hosting=datacenter) and am unable to find a way to insert the templates / charts usually available when clicking on "New Diagram" in the desktop / web application.

Am I going insane or is this a missing feature?

Gotta say that even as is draw.io is marvelous and enables so much more than miro or other solutions.

Stay safe out there!
--Noah

Edit: Adding File --> menu where File --> New Diagram would normally be to bring up the templates.
https://imgur.com/KpQBflI.png

**Comment by Preservation-Quest** [comment]
I emailed Draw.io support asking about this issue as well (wasn't aware there was a support desk.)

**Comment by mararad** [comment]
Hi,

Did you get our reply?

Thanks,

Marija

**Reply by Preservation-Quest** [reply]
> Hi,
> 
> Did you get our reply?
> 
> Thanks,
> 
> Marija

I did and it was extremely prompt and informative :) 

This feedback is made with complete respect for what I ackowledge appears simple but is actually an incredibly complex program especially with how great and intuitive the GUI is:

This feature omission is extremely frustrating because it removes the ability for a Team to create a diagram structure in the span of a few seconds as opposed to 10+ minutes. Exporting and importing from the desktop / web application is clunky in this case because you need to share your screen for everyone on the call to have feedback on the best diagram structure including modifications before importing it.

Very, very curious to me that being able to choose from diagram templates isn't seen as a core function of drawio.
